docker-for-mac/networking.md

```diff
@@ -103,14 +103,14 @@ docker run -d -p 80:80 --name webserver nginx
 To clarify the syntax, the following two commands both expose port `80` on the
 container to port `8000` on the host:
 
-		docker run --publish 8000:80 --name webserver nginx
-		docker run --p 8000:80 --name webserver nginx
+    docker run --publish 8000:80 --name webserver nginx
```

Comment:
If you are working on these, go ahead and use fences instead of indentation to delineate them. Thanks!